src/components/listing-card.js

```diff
@@ -28,7 +28,7 @@ class ListingCard extends Component {
   render() {
     return (
       <div className={`col-12 col-md-6 col-lg-4 listing-card${this.state.loading ? ' loading' : ''}`}>
-        <Link to={`/listing/${this.props.listingId}`}>
+        <Link to={`/listing/${this.props.address}`}>
```

Comment:
I would assume that this is expected to be `this.props.listingAddress`, but I don't actually see a change to `src/components/listing-grid`. 😕 Am I missing a commit?